CityOfDetroit/bloom
# 1314/copy updates unit group partners

## Issue

- Closes #1314
- Addresses #1314

## Description

- Removes hardcoded translations
- Fixes typo
- Renames the "edit programs" button to "edit communities"

## How Can This Be Tested/Reviewed?

1. Open partners, edit listing settings or create a new one
2. Scroll to the "community types" section
3. Make sure translations are correct
4. Click on the "add unit group" button
5. Check the "Affordable Unit Group Quantity" label and make sure there is no typo

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
